modules/prelude-literate-prgg.el

```diff
@@ -0,0 +1,78 @@
+;;; prelude-mind.el --- Emacs Prelude: Mind programming support.
+;;
+;; Author: Koustubh Sinkar
+;; Version: 1.0.0
+;; Keywords: literate programming
+
+;; This file is not part of GNU Emacs.
+
+;;; Commentary:
+
+;; Prelude configuration for literate programming
+
+;;; License:
+
+;; This program is free software; you can redistribute it and/or
+;; modify it under the terms of the GNU General Public License
+;; as published by the Free Software Foundation; either version 3
+;; of the License, or (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with GNU Emacs; see the file COPYING.  If not, write to the
+;; Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+;; Boston, MA 02110-1301, USA.
+
+;;; Code:
+(prelude-require-packages '(ob-cfengine3
```

Comment:
I'm wondering if we won't be going overboard by installing every possible package for literate programming by default. It might also be a good idea to add comments explaining the purpose of package like `ein`, `math-preview`, etc. Perhaps we can have some command with completion to install `ob-` packages on demand or something along those lines?